node/src/components/block_proposer/tests.rs

```diff
@@ -218,6 +260,47 @@ fn should_add_and_take_deploys() {
     assert!(deploys.contains(&deploy4.id()));
 }
 
+#[test]
+fn should_not_add_wasm_deploy_with_zero_gas_price() {
+    let creation_time = Timestamp::from(100);
+    let ttl = TimeDiff::from(Duration::from_millis(100));
+
+    let mut rng = crate::new_rng();
+    let deploy1 = generate_deploy(&mut rng, creation_time, ttl, default_gas_payment(), 0);
+    let mut proposer = create_test_proposer(0);
+    // pending
+    proposer.add_deploy_or_transfer(creation_time, *deploy1.id(), deploy1.deploy_type().unwrap());
+    assert_eq!(proposer.sets.pending.len(), 0, "there should be 0 pending deploys - the one added has a gas price (willingness to pay) of 0");
```

Comment:
You can break string literals with a backslash:
```suggestion
    assert_eq!(proposer.sets.pending.len(), 0, "there should be 0 pending deploys - \
        the one added has a gas price (willingness to pay) of 0");
```